avanza/avanza.py

```diff
@@ -2003,3 +2007,122 @@ def get_transactions(self,
+                            'name': str,
+                            'id': int,
+                            'urlParameterId': str,
+                        },
+                        'amount': {
+                            'decimalPrecision': int,
+                            'unit': str,
+                            'unitType': str,
+                            'value': int,
+                        },
+                        'availabilityDate': str,
+                        'comission': null|?,
+                        'currencyRate': null|?,
+                        'date': str,
+                        'description': str,
+                        'foreignTaxRate': null|?,
+                        'id': str,
+                        'instrumentName': str,
+                        'intraday': bool,
+                        'isin': str,
+                        'noteId': null|str
+                        'onCreditAccount': bool,
+                        'orderbook': {
+                            'isin': str,
+                            'currency': str,
+                            'flagCode': null|str,
+                            'name': str,
+                            'id': int,
+                            'type': str,
+                            'volumeFactor': int,
+                            'flagCode': null|str,
+                        },
+                        'priceInAccountCurrency': {
+                            'decimalPrecision': int,
+                            'unit': str,
+                            'unitType': str,
+                            'value': float,
+                        },
+                        'priceInTradedCurrency': {
+                            'decimalPrecision': int,
+                            'unit': str,
+                            'unitType': str,
+                            'value': float,
+                        },
+                        'settlementDate: str,
+                        'type': str,
+                        'volume': {
+                            'decimalPrecision': int,
+                            'unit': str,
+                            'unitType': str,
+                            'value': float,
+                        }
+                    },
+                ],
+            }
+        """
+        options = {}
+        options['maxElements'] = max_elements
+
+        if transaction_details_types:
+            options['transactionTypes'] = ','.join(transaction_details_types)
+        if transactions_from:
+            options['from'] = transactions_from.isoformat()
+        if transactions_to:
+            options['to'] = transactions_to.isoformat()
+        if isin:
+            options['isin'] = isin
+
+        return self.__call(
+            HttpMethod.GET,
+            Route.TRANSACTIONS_DETAILS_PATH.value,
+            options
+        )
+
+    def get_note_as_binary_blob(self, url_parameter_id: str, note_id: str):
```

Comment:
I see that the path `Route.NOTE_PATH` ends in .pdf, would it not be better to call this get_note_as_pdf?

Reply:
You are absolutely right.
Even `..._binary_blob` is duplicate since b in blob stands for binare... late night coding :)
Fixed now